Kernel/Net/TCPSocket.h

```diff
@@ -128,6 +128,10 @@ class TCPSocket final : public IPv4Socket {
     u32 packets_out() const { return m_packets_out; }
     u32 bytes_out() const { return m_bytes_out; }
 
+    static constexpr u32 maximum_duplicate_acks = 5;
```

Comment:
Add a comment describing how you conjured this magical constant? 😄

Reply:
Well, I've added _some_ comment to it. Not sure if that makes it better though. :thinking: